homework/index.js

```diff
@@ -28,14 +28,75 @@
     });
     return elem;
   }
+  // convert date time
+  function convertDataTime(dateTimeTxt) {
+    const dateTime = new Date(dateTimeTxt);
+    return dateTime.toLocaleString();
+  }
+  // sort repositories by name
+  function sortElements(a, b) {
```

Comment:
This is a case where you seem to need a comment because the function name is not clear enough. If you renamed the function to `sortRepositoriesByName` you could get rid of the comment. It is always better to let the code speak for itself.

Reply:
Thank you, it would be nicer to choose a clear name.